src/bournemouth/resources.py

```
@@ -16,17 +16,21 @@
 
 if typing.TYPE_CHECKING:  # pragma: no cover
-    from falcon.asgi import WebSocket
     from sqlalchemy.ext.asyncio import AsyncSession
 
     from .openrouter_service import OpenRouterService
 
+Struct = msgspec.Struct  # pyright: ignore[reportUntypedBaseClass]
+WebSocket = falcon.asgi.WebSocket  # pyright: ignore[reportUnknownArgumentType]
+MsgEncoder = msgspec_json.Encoder  # pyright: ignore[reportUnknownArgumentType]
+
 from .chat_service import (
```

Flake8 (7.2.0)

[error] module level import not at top of file

(E402)